### PR TITLE
Features content

### PR DESCRIPTION
**Pull request by lily4178993**
Leaders Week is a fictive summit for which I create this website. It follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) by [Cindy Shin in Behance](https://www.behance.net/adagio07).

This branch include:

**1- 2 pages:**
- The home page.
- The about page.
   
Each of these pages is UIs adaptable to different screen sizes 

**2- Interactions**

**2.1- Links**
- The home page has a link in the menu to the about page.
- The logo in the header links to the home page.

**2.2- Mobile menu**
When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.

**3- Dynamic page**
The section "Featured speakers" was created dynamically in JavaScript.

I kindly request someone to lend their expertise by examining this branch.
I would greatly appreciate any comments provided. Here is the [link](https://lily4178993.github.io/leaders-week/) of the live demonstration of the site.

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -0,0 +1,807 @@
...
+
+  .menu-box ul li a:hover,
+  .menu-box ul li a:focus {
+    transform: translateY(-3px);
+  }
+
+  .section-container {
+    max-width: 70%;
+    margin: auto;
+  }
```
Comment (AbelG101) [inline styles/styles.css:742]: - [ ] Ouch! this is close but you seem to be missing some spacing here as shown below (highlighted in yellow) in the original design. As a frontend developer, you have to have a good eye for small details like this so please make sure to work on them.
- Original design:
  image
- Your design:
  image

File: index.html [hunk 2]
```diff
@@ -4,10 +4,132 @@
...
+        <a href="index.html"><img src="images/logo-gc-1.png" alt="Leaders Week Summit 2023 Logo" class="hidden-item logo"></a>
+        <ul>
+            <li><a href="about.html" class="menu-item active">About</a></li>
+            <li><a href="index.html#Program" class="menu-item">Program</a></li>
+            <li><a href="#" class="menu-item">Join</a></li>
+            <li><a href="#" class="menu-item">Sponsor</a></li>
+            <li><a href="#" class="menu-item">News</a></li>
+            <li><a href="index.html" class="menu-item hidden-item red-bordered">LW Company</a></li>
+        </ul>
+    </nav>
```
Comment (AbelG101) [inline index.html]: - [ ] It seems like you forgot to add the top (dark) section of the navigation as shown in the original design. so please make sure to add them to make your design look more like the original.
- Original design:
  image
- Your design:
  image
- [ ] This applies to both the "Home" & "about" page. ✔

File: styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,807 @@
...
+  .footer-2 {
+    display: none;
+  }
+}
+
+/* ==== Mobile M - 375px ==== */
+@media screen and (width >= 375px) {
+  .section-container {
+    padding: 2em 1em;
+  }
```
Comment (AbelG101) [inline styles/styles.css:515]: - [ ] There seems to be a text overflow in the landing page of both the Home and About pages and this is making your UI look unprofessional so I highly advise you to work on them. 
- Here's how it looks:
  image
  image

File: styles/styles.css [hunk 4]
```diff
@@ -0,0 +1,807 @@
...
+  /* === COLORS === */
+  --color-dark: #272a31;
+  --color-darkOrange: #ec5242;
+  --color-grey: #d3d3d3;
+  --color-white: #fff;
+  --color-whitesmoke: #f9f9f9;
+
+  /* === FONTS FAMILY === */
+  --font-family-1: 'Lato', sans-serif;
+  --font-family-2: 'Dela Gothic One', cursive;
```
Comment (AbelG101) [inline styles/styles.css]: - [ ] I highly advise you to only use the given fonts as it creates discrepancies in relation to the original design. So Kindly replace the `--font-family-2` with `cocogoose`.
- [ ] And even though you've declared the "Lato" font here most of the texts in your page are using the browser's default font. 
- here's how it currently looks:
 firefox_XaZH4BML07

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @lily4178993 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Unique & personalized content ✔
-  Well-structured files ✔
- Clean and very effective code (HTML, CSS & JS) ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -4,10 +4,144 @@
...
+                    A joyful celebration in the value of openess and sharing,
+                    creating a positive change with people from over 10 countries
+                    is taking place in July, in Benin.
+                </p>
+                <div class="hero-event">
+                    <p class="date bold">2023.07.17(MON) ~ 21(FRI)</p>
+                    <p>@ Raddison House, Emporium and more</p>
+                </div>
+            </div>
+        </section>
```
Comment (mahbubul14) [inline index.html:55]: - [ ] You have implemented well according to the previous reviewer's suggestions. But you have changed some designs after that don't match the original design. Could you please work on font size, font weight, font family, and text color whatever is needed to match the original design? The description and the title of the summit should be stretched a little bit more to the right side. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
figma | yourdesign

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @lily4178993,

Great work on making the changes requested by the previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 5 above)
Comment (lily4178993) [reply index.html:55]: Please @mahbubul14 , can you explain a little bit what exactly I should change? I don't understand.

Comment (lily4178993) [reply index.html:55]: It's good now!

File: index.html [hunk 6]
```diff
@@ -4,10 +4,144 @@
...
+    <!-- MAIN -->
+    <main>
+        <!-- HERO SECTION -->
+        <section class="hero-section d-flex" id="Home">
+            <div class="section-container d-flex">
+                <div class="title-box d-flex">
+                    <h1 class="title bold">Leaders Week Summit 2023</h1>
+                    <h2 class="subtitle bold"><q>Hello! Sharing world</q></h2>
+                </div>
+                <p class="description">
```
Comment (roshan-bajgain) [inline index.html:45]: - [ ] [Optional] Everything is good only a small change is required so I am giving it an optional one. Please increase the font weight of the heading tag to match the original design of both the home page and the About page.

**Review by roshan-bajgain — APPROVED** [review]
Hi @lily4178993, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**